src/main/java/de/tinf22b6/dhbwhub/security/WebSecurityConfig.java

```diff
@@ -36,16 +36,22 @@ public class WebSecurityConfig {
             "/post/homepage-preview-posts/{id:\\d+}",
             "/post/post-thread/{id:\\d+}",
             "/post/popular-tags",
-            "/post/posts-by-tag/{tag}",
+            "/post/posts-by-tag/*",
             "/post/user-posts/{id:\\d+}",
-            "/post/posts-by-keyword/{keyword}",
-            "/post/posts-by-tag-keyword/{keyword}",
+            "/post/friend-posts/{id:\\d+}",
```

Comment:
`"/post/friend-posts/{id:\\d+}"` should not be public, I just deleted this from this config